src/App.tsx

```diff
@@ -1,11 +1,40 @@
 import React from "react";
 import Example from "./components/Example";
 import Layout from "./components/layout/Layout";
+import PollutedLocationCard from "./components/pollutedLocationCard/PollutedLocationCard";
+
+type PollutedLocationType = Partial<{
+  id?: string;
+  title?: string;
+  location: Partial<{
+    longitude?: number;
+    latitude?: number;
+  }>;
+  radius?: number;
+  severity?: string;
+  spotted?: string;
+  progress?: number;
+  notes?: string;
+}>;
 
 const App: React.FC = () => {
+  const pollutedLocation: PollutedLocationType = {
+    id: "uid-420-...",
```

Comment:
why not use a full guid?

```suggestion
    id: "a950ab10-d394-468b-b2ef-776135a65b96",
```

You can generate some random guids [here](https://www.guidgenerator.com/online-guid-generator.aspx).